bundle/language_en.properties

```diff
@@ -894,7 +902,8 @@ angal.medicalstock.cost.col
 angal.medicalstock.deletemovementsuccess.msg                                                           = Last movement successfully deleted.
 angal.medicalstock.discharge.btn                                                                       = Discharge
 angal.medicalstock.discharge.btn.key                                                                   = R
-angal.medicalstock.notpossibletodeletethismovementthemedicalhasbeenusedafterbeenreceivedinward.fmt.msg = Not possible to delete this movement: the medical {0} has already been used after been received in ward {1}. 
+angal.medicalstock.notpossibletodeletethismovementthemedicalhasbeenusedafterbeenreceivedinward.fmt.msg = Not possible to delete this movement: the medical {0} has already been used after been received in ward {1}.
+angal.medicalstock.notpossibletodeletethismovementbecauseitisrelatedtoaninventory.msg                    = Not possible to delete this movement because it is related to an inventory.
```

Comment:
```suggestion
angal.medicalstock.notpossibletodeletethismovementbecauseitisrelatedtoaninventory.msg                  = Not possible to delete this movement because it is related to an inventory.
```